var/spack/repos/builtin/packages/systemd/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Systemd(AutotoolsPackage):
```

Comment:
It looks like systemd uses Meson and should therefore be a `MesonPackage`. `configure` is only a convenience wrapper.

Reply:
I had started down this path initially but ran into issues with the meson build process and how systemd does things vs. how the ```MesonPackage``` expects things.  I'll revive my old draft that uses ```MesonPackage``` and see if I can switch it around for this PR.  Otherwise, if that remains problematic I'll spin the systemd work out into a separate PR.